java/wmts-layer/README.md

```diff
@@ -1,26 +1,36 @@
-# WMTS Layer
- Display a layer from a Web Map Tile Service. 
+# WMTS layer
+
+Display a layer from a Web Map Tile Service.
  
- ![WMTS Layer App](wmts-layer.png)
+![Image of WMTS layer](wmts-layer.png)
+
+## Use case
+
+WMTS services can have several layers. You can use Runtime to explore the layers available from a service. This would commonly be used to enable a browsing experience where users can choose which layers they want to display at run time.
 
 ## How to use the sample
-Simply run the app
+
+Pan and zoom to explore the WMTS layer, which is displayed automatically.
  
 ## How it works
-To display a `WmtsLayer` from a `WmtsService`:
  
 1. Create a `WmtsService` using the URL of the WMTS Service.
-1. After loading the WmtsService, get the list of `WmtsLayerInfos` from the service info: `service.getServiceInfo().getLayerInfos()`
+1. After loading the WMTS service, get the list of `WmtsLayerInfos` from the service info: `service.getServiceInfo().getLayerInfos()`
 1. For the layer you want to display, get the layer ID using `getLayerInfos().get(0).getId()`
```

Comment:
This sample doesn't seem to get the first layer id, it just gets the list and then goes straight to creating a wmts layer. notably, the comments say they get the first layer id as well.

Reply:
updated the readme to reflect what we do in the sample